src/fidesops/schemas/saas/saas_config.py

```diff
@@ -58,19 +58,12 @@ def check_reference_direction(
 
 
 class Strategy(BaseModel):
-    """General shape for swappable strategies (ex: auth, pagination, etc.)"""
+    """General shape for swappable strategies (ex: auth, processors, pagination, etc.)"""
 
     strategy: str
     configuration: Dict[str, Any]
 
 
-class PostProcessorStrategyData(BaseModel):
-    """Data class for postprocessor strategy"""
-
-    strategy: str
-    configuration: StrategyConfiguration
-
-
```

Comment:
OK I see you've removed this, but I think we're going to want to get the dynamic validation working on `strategy` and `configuration`.  You can pass in any strategy you want and any configuration you want. 

We have dynamic validation in a few places, for example ConnectionConfig secrets, we validate different pieces depending on the type of connection config.

Reply:
Reticketed here https://github.com/ethyca/fidesops/issues/266

Reply:
thanks @pattisdr !